### ISSUE TITLE
Wazuh Active Response execd not checking <white_list> IPs

### ISSUE THREAD

**Issue by dami-nicastro** [wazuh/wazuh#13969]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.2.7 | Wazuh manager | Manager/Agent | Packages/Sources | CentOS 7.9 |

Hi team:
 I have implemented this config in the Wazuh manager:
```
vi /var/ossec/etc/ossec.conf
...
<ossec_config>
  <global>
    <jsonout_output>yes</jsonout_output>
    <alerts_log>yes</alerts_log>
    <logall>no</logall>
    <logall_json>yes</logall_json>
    <email_notification>no</email_notification>
    <smtp_server>smtp.example.wazuh.com</smtp_server>
    <email_from>ossecm@example.wazuh.com</email_from>
    <email_to>recipient@example.wazuh.com</email_to>
    <email_maxperhour>12</email_maxperhour>
    <email_log_source>alerts.log</email_log_source>
    <agents_disconnection_time>10m</agents_disconnection_time>
    <agents_disconnection_alert_time>0</agents_disconnection_alert_time>
    <white_list>192.168.19.128</white_list>
  </global>
...
<active-response>
    <disabled>no</disabled>
    <command>firewall-drop</command>
    <rules_id>554</rules_id>
    <location>local</location>
    <timeout>120</timeout>
  </active-response>
...
```
In the agent with IP `192.168.19.128` I have the following `syscheck` configuration:
```
vi /var/ossec/etc/ossec.conf
...
<syscheck>
    <disabled>no</disabled>

    <!-- Frequency that syscheck is executed default every 12 hours -->
    <frequency>43200</frequency>

    <scan_on_start>yes</scan_on_start>

    <!-- Directories to check  (perform all possible verifications) -->
    <directories>/etc,/usr/bin,/usr/sbin</directories>
    <directories>/bin,/sbin</directories>
    <directories check_all="yes" realtime="yes">/root</directories>
...
```
When I create a file in `/root` I only should have the `syscheck` but I also have the `Active response` meaning the white listed IP is blocked though `firewall-drop`:
```
{
  "_index": "wazuh-alerts-4.x-2022.06.23",
  "_type": "_doc",
  "_id": "M3TCj4EBZceMcr98cokS",
  "_version": 1,
  "_score": null,
  "_source": {
    "input": {
      "type": "log"
    },
    "agent": {
      "ip": "192.168.19.128",
      "name": "centos-agent2",
      "id": "012",
      "labels": {
        "group": "Linux"
      }
    },
    "manager": {
      "name": "wazuh-man-4.2.1"
    },
    "data": {
      "origin": {
        "module": "wazuh-execd",
        "name": "node01"
      },
      "version": "1",
      "parameters": {
        "alert": {
          "syscheck": {
            "uname_after": "root",
            "mtime_after": "2022-06-23T05:51:23",
            "size_after": "0",
            "gid_after": "0",
            "mode": "realtime",
            "path": "/root/ar_test4",
            "sha1_after": "da39a3ee5e6b4b0d3255bfef95601890afd80709",
            "gname_after": "root",
            "uid_after": "0",
            "perm_after": "rw-r--r--",
            "event": "added",
            "md5_after": "d41d8cd98f00b204e9800998ecf8427e",
            "sha256_after": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
            "inode_after": "33879536"
          },
          "agent": {
            "ip": "192.168.19.128",
            "name": "centos-agent2",
            "id": "012",
            "labels": {
              "group": "Linux"
            }
          },
          "manager": {
            "name": "wazuh-man-4.2.1"
          },
          "rule": {
            "firedtimes": "1",
            "mail": "false",
            "level": "5",
            "pci_dss": [
              "11.5"
            ],
            "hipaa": [
              "164.312.c.1",
              "164.312.c.2"
            ],
            "tsc": [
              "PI1.4",
              "PI1.5",
              "CC6.1",
              "CC6.8",
              "CC7.2",
              "CC7.3"
            ],
            "description": "File added to the system.",
            "groups": [
              "ossec",
              "syscheck",
              "syscheck_entry_added",
              "syscheck_file"
            ],
            "id": "554",
            "nist_800_53": [
              "SI.7"
            ],
            "gpg13": [
              "4.11"
            ],
            "gdpr": [
              "II_5.1.f"
            ]
          },
          "location": "syscheck",
          "id": "1655974246.6221686",
          "decoder": {
            "name": "syscheck_new_entry"
          },
          "timestamp": "2022-06-23T05:50:46.599-0300",
          "full_log": "File '/root/ar_test4' added\nMode: realtime\n"
        },
        "extra_args": [],
        "program": "active-response/bin/firewall-drop"
      },
      "command": "add"
    },
    "rule": {
      "firedtimes": 1,
      "mail": false,
      "level": 3,
      "pci_dss": [
        "11.4"
      ],
      "tsc": [
        "CC6.1",
        "CC6.8",
        "CC7.2",
        "CC7.3",
        "CC7.4"
      ],
      "description": "Host Blocked by firewall-drop Active Response",
      "groups": [
        "ossec",
        "active_response"
      ],
      "id": "651",
      "nist_800_53": [
        "SI.4"
      ],
      "gpg13": [
        "4.13"
      ],
      "gdpr": [
        "IV_35.7.d"
      ]
    },
    "location": "/var/ossec/logs/active-responses.log",
    "decoder": {
      "name": "ar_log_json"
    },
    "id": "1655974247.6222385",
    "full_log": "2022/06/23 05:51:23 active-response/bin/firewall-drop: {\"version\":1,\"origin\":{\"name\":\"node01\",\"module\":\"wazuh-execd\"},\"command\":\"add\",\"parameters\":{\"extra_args\":[],\"alert\":{\"timestamp\":\"2022-06-23T05:50:46.599-0300\",\"rule\":{\"level\":5,\"description\":\"File added to the system.\",\"id\":\"554\",\"firedtimes\":1,\"mail\":false,\"groups\":[\"ossec\",\"syscheck\",\"syscheck_entry_added\",\"syscheck_file\"],\"pci_dss\":[\"11.5\"],\"gpg13\":[\"4.11\"],\"gdpr\":[\"II_5.1.f\"],\"hipaa\":[\"164.312.c.1\",\"164.312.c.2\"],\"nist_800_53\":[\"SI.7\"],\"tsc\":[\"PI1.4\",\"PI1.5\",\"CC6.1\",\"CC6.8\",\"CC7.2\",\"CC7.3\"]},\"agent\":{\"id\":\"012\",\"name\":\"centos-agent2\",\"ip\":\"192.168.19.128\",\"labels\":{\"group\":\"Linux\"}},\"manager\":{\"name\":\"wazuh-man-4.2.1\"},\"id\":\"1655974246.6221686\",\"full_log\":\"File '/root/ar_test4' added\\nMode: realtime\\n\",\"syscheck\":{\"path\":\"/root/ar_test4\",\"mode\":\"realtime\",\"size_after\":\"0\",\"perm_after\":\"rw-r--r--\",\"uid_after\":\"0\",\"gid_after\":\"0\",\"md5_after\":\"d41d8cd98f00b204e9800998ecf8427e\",\"sha1_after\":\"da39a3ee5e6b4b0d3255bfef95601890afd80709\",\"sha256_after\":\"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\",\"uname_after\":\"root\",\"gname_after\":\"root\",\"mtime_after\":\"2022-06-23T05:51:23\",\"inode_after\":33879536,\"event\":\"added\"},\"decoder\":{\"name\":\"syscheck_new_entry\"},\"location\":\"syscheck\"},\"program\":\"active-response/bin/firewall-drop\"}}",
    "timestamp": "2022-06-23T05:50:47.215-0300"
  },
  "fields": {
    "timestamp": [
      "2022-06-23T08:50:47.215Z"
    ]
  },
  "sort": [
    1655974247215
  ]
}
```

Thanks for your help!

**Comment by dami-nicastro** [comment]
We will review this and submit another issue
